utils/getProviders.js

```diff
@@ -0,0 +1,39 @@
+const amendOptions = require("./amendOptions");
+const amendProviders = require("./amendProviders");
+const filterProviders = require("./filterProviders");
+const fetchOembedProviders = require("./fetchOembedProviders");
```

Comment:
@raae this is a little silly. I couldn't for the life of me figure out why node wouldn't let me `const { ... } = require('./index')`.

Reply:
Yeah, I was very unfamiliar with node.js at the time and everything was in one large file, and I accepted a PR with this solution.

Its been on my todo list to get some help with the architecture of the whole thing after I got everything working, but never got around to it.  

Feel free to give me some advice :D

Reply:
It's not a bad structure at all. I'm more making excuses for why I couldn't include utils the same way you do elsewhere :)

Tried to keep with the paradigms of your project.